packages/notification-services-controller/CHANGELOG.md

```diff
@@ -7,11 +7,28 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.1]
+
+### Added
+
+- export `defaultState` for `NotificationServicesController` and `NotificationServicesPushController`. ([#4441](https://github.com/MetaMask/core/pull/4441))
+
+- export `NOTIFICATION_CHAINS_ID` which is a const-asserted version of `NOTIFICATION_CHAINS` ([#4441](https://github.com/MetaMask/core/pull/4441))
+
+- export `NOTIFICATION_NETWORK_CURRENCY_NAME` and `NOTIFICATION_NETWORK_CURRENCY_SYMBOL`. Allows consistent currency names and symbols for supported notification services ([#4441](https://github.com/MetaMask/core/pull/4441))
+
+- added `isPushIntegrated` optional env property (defaults to true) ([#4441](https://github.com/MetaMask/core/pull/4441))
```

Comment:
Looks good! Just had one last addition for clarity:
```suggestion
- add `isPushIntegrated` as an optional env property in the `NotificationServicesController` constructor (defaults to true) ([#4441](https://github.com/MetaMask/core/pull/4441))
```

Reply:
Merged!

Reply:
Silly question - but do we have a changelog "best practices" doc or notes? E.g. past tense, be descriptive, etc

Reply:
No that's a good point. Closest thing we have is this: https://github.com/MetaMask/core/blob/main/docs/reviewing-release-prs.md (especially section 5), but it could definitely be more comprehensive.